frontend/app/component/form/data_entry/polling_station_choice/PollingStationChoiceForm.tsx

```diff
@@ -38,17 +43,24 @@ export function PollingStationChoiceForm({ anotherEntry }: PollingStationChoiceF
 
   const handleSubmit = () => {
     if (pollingStationNumber === "") {
-      setShowAlert(true);
+      setAlert(INVALID_POLLING_STATION_ALERT);
       return;
     }
 
     const parsedStationNumber = parsePollingStationNumber(pollingStationNumber);
     const pollingStation = pollingStations.find((pollingStation) => pollingStation.number === parsedStationNumber);
+    const pollingStationStatus = electionStatuses.statuses.find((status) => status.id === pollingStation?.id);
+
+    if (pollingStationStatus?.status === "definitive") {
```

Comment:
It looks a bit weird to use pollingStationStatus.status, maybe we can rename the const?

Reply:
[a932acc](https://github.com/kiesraad/abacus/pull/367/commits/a932accef1b712d26fa3d2a86a502af6202dc2fd)